File: backend/src/main/java/com/desarrollo/raffy/dto/ReviewFromFrontToBackDTO.java
package com.desarrollo.raffy.dto;

import com.desarrollo.raffy.model.AwardAlignment;
import com.desarrollo.raffy.model.CommunicationRating;
import com.desarrollo.raffy.model.DeliveryStatus;

import lombok.Data;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Data

@Getter @Setter
@NoArgsConstructor

public class ReviewFromFrontToBackDTO {
    private String email;
    private int score;
    private DeliveryStatus delivery;
    private AwardAlignment awardAlingment;
    private CommunicationRating communicationRating;
    private String comment;
}

File: frontend/src/app/models/review/reviewFromFrontToBackDTO.ts
import { AwardAlignment } from "./AwardAlignment";
import { CommunicationRating } from "./CommunicationRating";
import { DeliveryStatus } from "./DeliveryStatus";

export interface reviewFromFrontToBackDTO {
    email: string,
    delivery: DeliveryStatus | null,
    awardAlingment: AwardAlignment | null,
    communicationRating: CommunicationRating | null,
    score: number,
    comment: string
}